contentreview-impl/impl/src/java/org/sakaiproject/turnitin/util/TurnitinLTIUtil.java

```diff
@@ -87,6 +89,11 @@ public void init() {
 		if(StringUtils.isEmpty(turnitinSite)){
 			log.error("Turnitin global site property does not exist or is wrongly configured.");
 		}
+		turnitinReportsSite = serverConfigurationService.getString("turnitin.reports.lti.site", "!turnitin_reports");
```

Comment:
I think it would be good if we didn't require users to create multiple LTI tools and have to keep the configuration in sync between the two.